Contest_API.md

```diff
@@ -773,12 +787,22 @@ JSON elements of group objects:
 | icpc\_id | string  | no        | yes       | CCS        | external identifier from ICPC CMS                                        |
 | name     | string  | yes       | no        | CCS        | name of the group                                                        |
 | type     | string  | no        | yes       | CCS        | type of this group                                                       |
-| hidden   | boolean | no        | yes       | CCS        | if group should be hidden from scoreboard. Defaults to false if missing. |
+| hidden   | boolean | no        | yes       | CCS        | if the group is to be excluded from the [scoreboard](#scoreboard). Defaults to false if missing. |
 
 #### Access restrictions at WF
 
 No access restrictions apply to a GET on this endpoint.
 
+#### Known group types
+
+The list below contains standardized identifiers for known group
+types. These identifiers should be used when the purpose
+of a group matches.
+
+| Type  | Description
+| ----- | ---------------------------------
+| site  | A physical location where teams are competing, e.g. the "Hawaii site". Teams generally should not be in more than one group of this type.
```

Comment:
Remove "generally" as "should" already expresses the same. I would even prefer changing that into a "must" though.

Reply:
I very much agree with you, but isn't this the very thing you argued against earlier?

> I don't like it so much to attach special meaning to a particular group type `site`: that this one must be a partition of all teams.

Maybe I misunderstood your argument there?

Reply:
Well, I prefer making this a `must` over `should`, but I think it's even _better_ to make this a requirement encoded somehow in the specification of the `type` attribute, not on this specific type "site".